rekcurd/utils/__init__.py

```diff
@@ -70,23 +70,16 @@ def __init__(self, label: PredictLabel, score: PredictScore, option: dict = None
 
 
 class EvaluateResult:
-    def __init__(self, num: int = None, accuracy: float = None,
-                 precision: List[float] = None, recall: List[float] = None,
-                 fvalue: List[float] = None, option: Dict[str, float] = {}):
-        if num is None:
-            self.num = 0
-            self.accuracy = 0.0
-            self.precision = [0.0]
-            self.recall = [0.0]
-            self.fvalue = [0.0]
-            self.option = {}
-        else:
-            self.num = num
-            self.accuracy = accuracy
-            self.precision = precision
-            self.recall = recall
-            self.fvalue = fvalue
-            self.option = option
+    def __init__(self, num: int, accuracy: float, precision: List[float],
+                 recall: List[float], fvalue: List[float], label: List[PredictLabel],
+                 option: Dict[str, float] = {}):
```

Comment:
BTW, I think `option` type is just `dict` not `Dict[str, float]`...?

Reply:
@keigohtr 
Now gprc spec is `map<string, float>` [here](https://github.com/rekcurd/grpc-proto/blob/master/rekcurd.proto#L217) because I just expected the option is just used for additional metrics (e.g. accuracy in a specific condition), so it needs to be `Dict[str, float]`

(it will be OK if we change grpc spec)

Reply:
It's OK at the moment. 

Could you make an issue for this?
[Additional metrics](https://github.com/rekcurd/grpc-proto/blob/master/rekcurd.proto#L217) must be a general field for ML evaluation. Current spec specifies our use-case, so we need to fix it.

Reply:
I did it! https://github.com/rekcurd/rekcurd-python/issues/35